### PR TITLE
Cust 2032 correct header styling

### PR DESCRIPTION
**Pull request by darushka**
This PR solves 3 issues:
1.https://whentomanage.atlassian.net/browse/CUST-2007 - a softer drop shadow for peach-cards
2.https://whentomanage.atlassian.net/browse/CUST-2032 - correct text styling for inviting a new user
3.https://whentomanage.atlassian.net/browse/CUST-2029 - correct message box margin

### CODE REVIEW

File: client/app/common/directives/message/message.scss [hunk 1]
```diff
@@ -37,13 +39,12 @@ peach-message {
   }
 }
 
-md-content {
+peach-modal-content {
```
Comment (jsmylnycky) [inline client/app/common/directives/message/message.scss]: My only question here is if the message is being QA'd when not in a peachModal but inside of mdContent, and if it's still correct.

Comment (darushka) [inline client/app/common/directives/message/message.scss]: @jsmylnycky 
I have added default margin 30px (bottom 0px) to all messages. Those which are on peachModal have no margin. I tested on Config and Cake apps. Looks correct. But if you don't want to apply default margin to all messages, I can make a small refactoring.

Comment (jsmylnycky) [inline client/app/common/directives/message/message.scss]: If it passes QA this way, its fine by me :)

**Comment by jsmylnycky** [comment]
:+1:

**Comment by rynomars** [comment]
:+1:

**Comment by jsmylnycky** [comment]
@darushka It has been decided to cancel CUST-2007.  Can you please revert those changes out of this PR?  Thanks

**Comment by jsmylnycky** [comment]
peach.css v0.4.7

**Pull request closed without merging**